### DISCUSSION TITLE
API Portal: deprecated endpoint

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#201] [Q&A] [answered]
Hi!

If we use the API Portal, can we mark an endpoint as deprecated?

Thanks

**Comment by Danielecina** [comment, marked as answer]
Hi @giovf96, maybe you can use on your schema a specific tag: ['deprecated'] for each API that you want to deprecate, via API-portal you see each API grouped in a section deprecated. What do you think? 

```js
const schema = {
  tags: ['deprecated'],
  response: {...}
  ...other
}
```

On your API portal you will see...
![Schermata 2023-02-22 alle 17 19 53](https://user-images.githubusercontent.com/22480239/220688951-ac4001e7-be60-4d89-8183-f6bc78600b23.png)

**Reply by flower-of-the-bridges** [reply]
Hi @Danielecina @fredmaggiowski 

Thanks for the insights, I'll try to take a look to the repo! Meanwhile I'll use the tip provided by @Danielecina to achieve the result.

Thanks!

**Comment by fredmaggiowski** [comment]
The OAS standard `deprecated` flag is not supported by API Portal, in the sense that there is no clear UI that gives feedback to the user about API deprecation.

You could either to as suggested by @Danielecina or open an issue (or a PR, even better ❤️ ) in this repository: https://github.com/mia-platform/api-explorer
